src/protobuf.cpp

```diff
@@ -95,3 +104,15 @@ void ProtoBuffer::commitMessage(protobuf_mark_t mark) {
     }
     _data[mark - 1] = (unsigned char) message_length;
 }
+
+void ProtoBuffer::appendRepeated(bool b) {
+    putVarInt((u32) b);
+}
+
+void ProtoBuffer::appendRepeated(u32 n) {
+    putVarInt(n);
+}
+
+void ProtoBuffer::appendRepeated(u64 n) {
+    putVarInt(n);
+}
```

Comment:
I see no sense in all these extra functions. Why not just call `putVarInt` directly where needed?

Reply:
See comment above, I'll remove it

Reply:
93b4189c57119791275cf6bebb17571be243e911